src/shared/helpers/replace-at.ts

```diff
@@ -0,0 +1,5 @@
+export function replaceAt(array, index, value) {
```

Comment:
Does this need a unit test?

Reply:
Thanks for pointing it out, this should be deleted.